katalogus/templates/plugin_settings_add.html

```diff
@@ -7,15 +7,40 @@
   <main id="main-content">
     <section>
       <div>
-        <h1>{% translate 'Add setting - ' %}{{ setting_name }}</h1>
+        <h1>
+          {% if form.fields|length == 1%}
+            {% translate 'Add setting' %} {{ setting_name }}
+          {% else %}
+            {% translate 'Add settings' %}
+          {% endif %}
+        </h1>
         <p class="emphasized">
+          {% if form.fields|length == 1%}
           {% translate 'Setting' %}
+          {% else %}
+            {% translate 'Settings' %}
+          {% endif %}
         </p>
         <div class="form-view">
           <form novalidate method="post" class="horizontal-view help">
             {% csrf_token %}
             {% include 'partials/form/fieldset.html' with legend=name fields=form %}
-            <button type="submit">{% translate 'Add setting' %}</button>
+            <div class="horizontal-view">
+              <button type="submit" class="ghost" name="add">
+                {% if form.fields|length == 1%}
+                  {% translate 'Add setting' %}
+                {% else %}
+                  {% translate 'Add settings' %}
+                {% endif %}
```

Comment:
Please use [block translate](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#std:templatetag-blocktrans) for this

Reply:
Thanks for pointing this out, didn't know this existed.